test/powershell/Language/Scripting/NativeExecution/NativeCommandArguments.Tests.ps1

```diff
@@ -5,7 +5,12 @@ param()
 
 Describe "Behavior is specific for each platform" -tags "CI" {
     It "PSNativeCommandArgumentPassing is set to 'Windows' on Windows systems" -skip:(-not $IsWindows) {
-        $PSNativeCommandArgumentPassing | Should -Be "Windows"
+        if ([Version]::TryParse($PSVersiontable.PSVersion.ToString(), [ref]$null)) {
```

Comment:
This should be changed to check `$PSVersiontable.PSVersion.PreReleaseLabel`.

Reply:
@daxian-dbw  i think this should probably be different from the way that the code determines that it's a preview, unless you think it's going to be incorrect. I would rather use a different methodology so it's not subject to the same errors. If I use the same way in the code, there's a lower chance of finding inconsistencies.